Docker/docker-compose.yml

```diff
@@ -20,7 +20,10 @@ services:
         max-size: "200k"
         max-file: "10"
   adaguc-server:
-    image: openearth/adaguc-server
+    build:
+      context: ..
+      dockerfile: ./Dockerfile
```

Comment:
Works without this line, providing the `context` should be enough.

And we probably want to revert this.